exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/DrillSqlWorker.java

```diff
@@ -183,4 +184,23 @@ private static PhysicalPlan getQueryPlan(QueryContext context, String sql, Point
 
     return handler.getPlan(sqlNode);
   }
+
+  //Wrap with Auto Limit
+  private static SqlNode wrapWithAutoLimit(SqlConverter parser, QueryContext context, String sql) {
+    SqlNode sqlNode = parser.parse(sql);
+    if (context.isAutoLimitEnabled() && sqlNode.getKind().belongsTo(SqlKind.QUERY)) {
```

Comment:
The ```context.isAutoLimitEnabled()``` is now checked before method invocation and may be removed here:
```suggestion
    if (sqlNode.getKind().belongsTo(SqlKind.QUERY)) {
```

Reply:
Done